src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -161,14 +161,18 @@ class Statistics:
     def __init__(self, metrics: Metrics):
         # iterate through Metrics to calculate statistics and set attributes
         self._metrics = metrics
+        self._stats_dict: Dict = {}
         for attr, data in metrics.data.items():
             attr = metrics.get_base_name(attr)
+            self._stats_dict[attr] = {}
+            self._add_units(attr)
             data = self._preprocess_data(data, attr)
             if data:
                 self._calculate_mean(data, attr)
-                self._calculate_percentiles(data, attr)
-                self._calculate_minmax(data, attr)
-                self._calculate_std(data, attr)
+                if not self._is_throughput_field(attr):
```

Comment:
Why are you checking if the metric is throughput field or not?

Reply:
Because if its not a throughput field, I dont want to calculate more than the average.